compiler-plugin/src/main/scala/scalapb/compiler/GrpcServicePrinter.scala

```diff
@@ -163,6 +163,7 @@ final class GrpcServicePrinter(service: ServiceDescriptor, implicits: Descriptor
       |    .setSampledToLocalTracing(true)
       |    .setRequestMarshaller(${marshaller(method.inputType)})
       |    .setResponseMarshaller(${marshaller(method.outputType)})
+      |    .setSchemaDescriptor(new _root_.scalapb.grpc.ConcreteProtoMethodDescriptorSupplier(${service.getFile.fileDescriptorObjectFullName}.javaDescriptor, ${service.javaDescriptorSource}, ${method.javaDescriptorSource}))
```

Comment:
Please change to `_root_.scalapb.grpc.ConcreteProtoMethodDescriptorSupplier.fromMethodDescriptor(...)` and add the appropriate method in the companion object that instantiates this class (using the MethodDescriptor alone, see other comment). The reason for not calling the constructor directly here is that over time signatures evolve and it's easier to achieve binary and source compatibility if signatures that may somewhat change have names.